### ISSUE TITLE
Auto-Linking framework not found TSLocationManager

### ISSUE THREAD

**Issue by JackCharb** [transistorsoft/react-native-background-geolocation#520]
## Environment
* Plugin version: 2.12.2
* Platform: iOS
* Xcode version: 9.4.1
* React Native version: 0.55.4

## Expected Behavior
Application should load in iOS device simulator.

## Actual Behavior
Build fails and app cannot be run on a simulated device.

## Steps to Reproduce
1. Follow install and linking instructions form react-native-background-geolocation readme
2. Run command `sudo react-native run-ios`

## Context
I was trying to build the application for testing in Xcode simulator. I have tried solutions associated with similar issues (such as reinstalling and link react-native-background-fetch), but to no avail.

## Debug logs
ld: warning: Auto-Linking framework not found TSLocationManager
Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_TSBackgroundFetch", referenced from:
      objc-class-ref in libRNBackgroundFetch.a(RNBackgroundFetch.o)
  "_OBJC_CLASS_$_TSConfig", referenced from:
      objc-class-ref in libRNBackgroundGeolocation.a(RNBackgroundGeolocation.o)
  "_OBJC_CLASS_$_TSCurrentPositionRequest", referenced from:
      objc-class-ref in libRNBackgroundGeolocation.a(RNBackgroundGeolocation.o)
  "_OBJC_CLASS_$_TSGeofence", referenced from:
      objc-class-ref in libRNBackgroundGeolocation.a(RNBackgroundGeolocation.o)
  "_OBJC_CLASS_$_TSLocationManager", referenced from:
      objc-class-ref in libRNBackgroundGeolocation.a(RNBackgroundGeolocation.o)
  "_OBJC_CLASS_$_TSWatchPositionRequest", referenced from:
      objc-class-ref in libRNBackgroundGeolocation.a(RNBackgroundGeolocation.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

** BUILD FAILED **

**Comment by christocracy** [comment]
You haven't got the plugin installed correctly.  You missing the `TSLocationManager.framework`

**Comment by JackCharb** [comment]
I will try installing with the manual instructions later today and follow some of the advice I found in [this issue](https://github.com/transistorsoft/react-native-background-fetch/issues/46) that seems to be related. I will update with results as soon as I can.

**Comment by JackCharb** [comment]
I was able to fix this problem following the [manual installation instructions](https://github.com/transistorsoft/react-native-background-geolocation/blob/master/docs/INSTALL-IOS.md). In my case the particular fix was adding frameworks under the "Link Binary With Libraries" tab and then setting the appropriate framework search paths. I also had to move the RNBackgroundFetch+AppDelegate.m to its appropriate place.
